Roles/Impostor/Eraser.cs

```diff
@@ -101,7 +100,7 @@ public static void AfterMeetingTasks()
         {
             var player = Utils.GetPlayerById(pc);
             if (player == null) continue;
-            player.RpcSetCustomRole(CustomRolesHelper.GetErasedRole(player.GetCustomRole()));
+            player.RpcSetCustomRole(CustomRolesHelper.GetErasedRole(player.GetCustomRole().GetRoleTypes(), player.GetCustomRole()));

```

Comment:
The code produces ### GetRoleTypes(), accordingly returns the player’s ### RoleTypes and, depending on the role type, returns a vanilla role (naturally, it cannot erase neutral roles)